index.html

```diff
@@ -2,35 +2,81 @@
             <div class="header-bg"></div>
+            <div class="header-text"><h1>Marcin Łaszczyński</h1><p><strong class="title">Tile Installer</strong></p></div>
         </div>
     </header>
     <nav class="nav">
-
         <ul>
-            <li class="nav-item"><i class="fa-solid fa-envelope"></i><a href="index.html">mar.laszcz@wp.pl</a></li>
-            <li class="nav-item"><i class="fa-solid fa-phone"></i><a href="service.html">506055400</a></li>
-            <li class="nav-item"><i class="fa-brands fa-github"></i><a href="https://github.com/MarcinL09">github.com/MarcinL09</a></li>
-            <li class="nav-item"><a href="contact.html">kontakt</a></li>
+            <li class="nav-item"><i class="fa-solid fa-envelope"></i><a href="mailto: mar.laszcz@wp.pl">mar.laszcz@wp.pl</a></li>
+            <li class="nav-item"><i class="fa-solid fa-phone"></i><a href="service.html">505 293 559</a></li>
+            <li class="nav-item"><i class="fa-brands fa-github"></i><a href="https://github.com/MarcinL09" target="_blank">github.com/MarcinL09</a></li>
         </ul>
-
-
-
     </nav>
     <main>
-        <h3 class="exp">Experience</h3>
+        <h3>Experience</h3>
+        <section class="exp">
+            <div class="left-side"><p>2020 - today</p></div>
+            <div class="right-side"><p><strong>Tile installer</strong> at <strong>Vivia Service</strong><br>Laying and setting tiles to create decorative wall, mural and floor designs. Cutting and fitting tiles around obstacles and openings using hand and power cutting tools. Preparing, measuring and marking surfaces to be covered.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2018 - 2020</p></div>
+            <div class="right-side"><p><strong>Assembly operator</strong> at <strong>Polentex</strong><br>Following operative guidelines to build a product for a company based on blueprints and instructions.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2013 - 2015</p></div>
+            <div class="right-side"><p><strong>Assembly operator</strong> at <strong>Xylem (Let’s Solve Water)</strong><br>
+                Working to strict safety and quality requirements, help to achieve daily
+                production requirements in terms of quality and quantity to meet customer
+                expectations and requirements, optimising efficiency and maintaining
+                operational excellence. Perform many operations within the process with
+                some supervision.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2012 - 2013</p></div>
+            <div class="right-side"><p><strong>Motorboats builder</strong> at <strong>Delphia Yachts Kot</strong><br>Reading and interpreting blueprints to build ships and boats. Makes repairs as necessary. Uses a variety of tools, instruments, and materials to create the boat.</p></div>
```

Comment:
I think you mixed the English tenses here a little. You've started this paragraph with "Reading and interpreting blueprints", so we should make small adjustments:

"Makes repairs as necessary" -> "Making repairs as necessary"
"Uses a variety of tools" -> "Using a variety of tools"